II.pass-tests/01-add-one/add-one.js

```diff
@@ -1,3 +1,13 @@
-module.exports = function (numbers) {
+module.exports = function(numbers) {
+  var newArray = [];
+  for (var i = 0; i < numbers.length; i++) {
+    newArray.push(numbers[i] + 1);
+  }
```

Comment:
Nice work :+1: 
Is there another way to implement this function? Can you think of a way using `map`?